zilliqa/src/consensus.rs

```diff
@@ -108,6 +111,64 @@ impl From<Hash> for MissingBlockError {
     }
 }
 
+#[derive(Debug, Clone)]
```

Comment:
this just contains the minimal set of info rather than the full TX so we can define a custom ordering and don't have to copy the tx in to new_transactions and new_transactions_priority

Reply:
I reckon there must be a cleaner way of representing this. I'll have a think/research and come back to this PR.